packages/paste-core/components/timeline/__tests__/index.spec.tsx

```diff
@@ -0,0 +1,141 @@
+import { render } from "@testing-library/react";
+import { UserIcon } from "@twilio-paste/icons/esm/UserIcon";
+import * as React from "react";
+
+import { Timeline, TimelineItem } from "../src";
+import { TimelineItemGroup } from "../src/TimelineItemGroup";
+
+const ExampleTimeline: React.FC<{
+  element?: string;
+}> = ({ element = "TIMELINE" }) => (
+  <Timeline element={element} data-testid="timeline">
+    <TimelineItem data-testid="timelineItem" element={`${element}_ITEM`} title="Start" timestamp="2018-03-01:10:00">
+      Event details
+    </TimelineItem>
+
+    <TimelineItem title="Inprogress" timestamp="2018-03-01:12:00">
+      Event details
+    </TimelineItem>
+
+    <TimelineItem title="Complete" timestamp="2018-03-01:14:00">
+      Event details
+    </TimelineItem>
+
+    <TimelineItem data-testid="timelineItemNoTimestamp" title="Item without timestamp">
+      Event details
+    </TimelineItem>
+
+    <TimelineItem title="Icon Item" timestamp="2018-03-01:12:00" icon={UserIcon}>
+      Event details
+    </TimelineItem>
+
+    <TimelineItemGroup timestamp="2018-03-01" data-testid="timelineItemGroup" element={`${element}_ITEM_GROUP`}>
+      <TimelineItem title="Inprogress" timestamp="12:00">
+        Event details
+      </TimelineItem>
+
+      <TimelineItem title="Complete" timestamp="14:00">
+        Event details
+      </TimelineItem>
+
+      <TimelineItem title="Item without timestamp">Event details</TimelineItem>
+    </TimelineItemGroup>
+
+    <TimelineItem title="Start" timestamp="2018-03-01:10:00" collapsible data-testid="timelineItemCollapsible">
+      Event details
+    </TimelineItem>
+
+    <TimelineItem
+      title="Item without timestamp"
+      collapsible
+      collapsibleHeading="custom heading"
+      data-testid="timelineItemCollapsibleNoTimestamp"
+    >
+      Event details
+    </TimelineItem>
+
+    <TimelineItem
+      title="Item with timestamp and collapsibleHeading"
+      timestamp="2018-03-01:10:00"
+      collapsible
+      collapsibleHeading="custom heading"
+      data-testid="timelineItemCollapsibleHeading"
+    >
+      Event details
+    </TimelineItem>
+  </Timeline>
+);
+
+describe("Timeline", () => {
+  it("should render", () => {
+    const { getByTestId } = render(<ExampleTimeline />);
```

Comment:
suggestion(optional): you could move this line outside of the `it` blocks so that you only have to render it once as every block in this describe is using the same line

Reply:
for some reason it didnt work. It works for first test, and then fails for all others if I do this